utils/parse.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+/* eslint-disable no-console */
```

Comment:
let's use overrides in the eslintrc instead of inline comments for these

Reply:
no - eslintrc's "overrides" can apply to specific globs of files, so you can indicate that `utils/*`, or `utils/parse.js`, etc, should have the no-console rule set to "off".

Reply:
TIL.

How's this?

- https://github.com/import-js/eslint-plugin-import/pull/2983/commits/d6907f795fcd7732d3c866d1646929316d796fcc

Reply:
looks great, thanks!